components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -197,7 +197,7 @@ export const SettingsSidebarButton = styled<SettingsSidebarButtonProps, 'button'
 export const SettingsFeatureBody = styled<{}, 'section'>('section')`
   padding: 10px 0;
   height: 360px;
-  overflow-y: auto;
+  overflow-y: overlay;
```

Comment:
The `overlay` value for `overflow` is deprecated. Should we be concerned about future support?

Reply:
I know it is in Firefox, but afaik there weren't plans to deprecate it in Webkit browsers. Is it on track for deprecation in Chrome?

Reply:
Hmm... I'm not exactly sure if it's only deprecated in Firefox, or if it's being deprecated across the board. I'm not seeing it in the spec for overflow, though: https://www.w3.org/TR/css-overflow-3/#overflow-properties

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#Values

Reply:
Looks like deprecation is in discussion, but I'm not seeing anything definitive. Don't think this should be a blocker. Sorry to distract.
https://bugs.chromium.org/p/chromium/issues/detail?id=554361#c19